t/v1.t

```diff
@@ -417,3 +417,45 @@ false
 --- response_body
 true
 false
+
+=== TEST 22: operator ~*
+--- config
+    location /t {
+        content_by_lua_block {
+            local expr = require("resty.expr.v1")
+            local ex = expr.new({
+                {"x", "~*", "A"}
+            })
+
+            ngx.say(ex:eval({x = 'a'}))
+            ngx.say(ex:eval({x = 'A'}))
+            ngx.say(ex:eval({x = 'b'}))
+            ngx.say(ex:eval({x = ''))
```

Comment:
Missing '}'.
You can install the test framework like this: https://github.com/api7/lua-resty-expr/blob/e58b38e1d654f5c4814c23d22985ee9dd843dfeb/.github/workflows/build.yml#L24

Reply:
fix